coordinator/src/main/scala/filodb.coordinator/queryplanner/SingleClusterPlanner.scala

```diff
@@ -332,7 +332,7 @@ class SingleClusterPlanner(dsRef: DatasetRef,
     val shardsToHit = if (shardColumns.toSet.subsetOf(lp.labelConstraints.keySet)) {
       shardsFromFilters(filters, options)
     } else {
-      mdNoShardKeyFilterRequests.increment()
+      mdNoShardKeyFilterRequests.withoutTags().increment()
```

Comment:
Check if `withoutTags` is correct - we actually need tags. If needed, then move to declaration.